utils/update_checkout/update-checkout-config.json

```diff
@@ -65,7 +65,7 @@
                 "llbuild": "main",
                 "swift-tools-support-core": "main",
                 "swiftpm": "main",
-                "swift-argument-parser": "0.3.0",
+                "swift-argument-parser": "0.4.0",
                 "swift-crypto": "1.1.4",
```

Comment:
We have to bump this to `1.1.5` too

Reply:
@yim-lee created another PR for it.

Reply:
https://github.com/apple/swift/pull/36314